src/main/java/com/advanon/pdfsignatures/DigestException.java

```diff
@@ -0,0 +1,9 @@
+package com.advanon.pdfsignatures;
+
+public class DigestException extends Exception {
+  private static final long serialVersionUID = 8086005637439302250L;
```

Comment:
VS code asked me to generate serialVersionUID for Serializable objects, so I assumed that is a kind of a best practice and generated this UID with `serialver` to suppress the highlight. But since we are not serializing anything, I guess this UID can be removed, right?

Reply:
You can remove all of them and java will use one anyway if needed